static/app/views/starfish/queries/useSpanMetrics.tsx

```diff
@@ -50,14 +50,15 @@ function getEventView(
   search: MutableSearch | undefined,
   fields: string[] = [],
   sorts: Sort[] = [],
-  pageFilters: PageFilters
+  pageFilters: PageFilters,
+  dataset: DiscoverDatasets = DiscoverDatasets.SPANS_METRICS
```

Comment:
🤔 why do this, if this is the `useSpanMetrics` hook? It should always be set to use the `SPANS_METRICS` dataset, no?

Reply:
Yeah you're right, this wasn't supposed to be here tbh, i was starting to do this for the non series hooks but that will be in another PR, i'll revert it.